src/afl-gotcpu.c

```diff
@@ -214,7 +214,13 @@ int main(int argc, char **argv) {
   #if defined(__linux__)
       if (sched_setaffinity(0, sizeof(c), &c)) {
 
-        PFATAL("sched_setaffinity failed for cpu %d", i);
+        const char *error_code = "Unkown error code";
+        if (errno == EFAULT) error_code = "EFAULT";
```

Comment:
switch case for legibility?

Reply:
dunno what you mean :)

Reply:
Like
```c
switch (errno) {
    case EFAULT:
        error_code = "EFAULT";
        break;
    case ...
```

Reply:
was a PR, nobody is using the tool really ... yes that would be nicer, but I like to invest my time better :)